distribution/common/src/main/resources/etc/profiles/ha.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
❓ do you see any maintenance issues (for example, keeping them in sync) between the DDF and Alliance distributions?

Reply:
Absolutely! This is actually a big tech debt item. I will create a spike ticket to look into alternative solutions and link it here.

Reply:
Spike ticket to investigate alternative solutions in DDF (since that's where the solution will be implemented): https://codice.atlassian.net/browse/DDF-3395